data/network/src/commonMain/kotlin/social/androiddev/common/network/model/Application.kt

```diff
@@ -17,9 +17,9 @@ import kotlinx.serialization.Serializable
  */
 @Serializable
 data class Application(
-    val id: String,
+    val id: String?=null,
```

Comment:
nullable `id` doesn't make sense to me... are you re-using this model for a different call which has nullable fields for some reason? 

If so, I would suggest creating a separate model for that case, so we don't lose this type safety. Otherwise, we will need to manually enforce the type safety on each call instead of a pure self-documenting model.

Reply:
Was rather weird to me too but the api returned null for an ID for some of the applications. We crashed on first fetch of my feed in androiddev.social. I'll make a localID for this for us and analyze why it is null. I'm not clear what Application even does for now

Reply:
double checked docs and ID does not exist as a field https://docs.joinmastodon.org/entities/Application/
We can do a local ID in db